tests/e2e_logging.py

```diff
@@ -656,8 +662,11 @@ def test_historical_receipts(network, args):
             r = first_receipt.json()["receipt"]
```

Comment:
It'd be great to add the receipt verification to `tutorial.py` and include it in https://microsoft.github.io/CCF/main/use_apps/verify_tx.html#transaction-receipts and/or https://microsoft.github.io/CCF/main/use_apps/python_tutorial.html (separate PR).